app/views/privacy_policies/show.html.erb

```diff
@@ -1,3 +1,8 @@
+<%= render GovukComponent::BackLink.new(
+      text: 'Back',
+      href: :back,
```

Comment:
Why is this one different?

Reply:
Would it not be simpler to set request.referer on the session as part of show? Rather than running this before every action on every controller?

Reply:
That was my first thought, but it doesn't fit every use case. When we update the form, the request.referer becomes "/cookies" ...

Reply:
I think personally I would prefer containing this all to the cookies controller (by not updating the session if the referer is `/cookies`) but I don't feel strongly enough to insist you change this

Reply:
I don't mind changing it, but it seems more complicated than it looks to contain it all in the cookies controller. I  you have a working suggestion that will fit all use cases, happy to implement?